src/main/java/com/esri/samples/na/find_route/FindRouteSample.java

```diff
@@ -109,10 +110,10 @@ public void start(Stage stage) throws Exception {
           routeGraphic = new Graphic(shape, new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, BLUE_COLOR, 2));
           routeGraphicsOverlay.getGraphics().add(routeGraphic);
 
-          // get route street names
-          route.getDirectionManeuvers().stream().flatMap(mvr -> mvr.getManeuverMessages().stream()).filter(ms -> ms
-              .getType().equals(DirectionMessageType.STREET_NAME)).forEach(st -> directionsList.getItems().add(st
-                  .getText()));
+          // get the direction text for each maneuver
+          for(DirectionManeuver step : route.getDirectionManeuvers()) {
```

Comment:
Should be `for (DirectionManeuver step : route.getDirectionManeuvers())` with a space between for and the bracket. I'm sure you could replace the loop with some stream based thing though.